packages/client/hmi-client/src/workflow/tera-optimize-chart.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+	<div class="simulate-chart">
+		<div class="row">
+			<Chart
+				type="bar"
+				:width="chartSize.width"
+				:height="chartSize.height"
+				:data="chartData"
+				:options="chartOptions"
+			/>
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { ref, computed, watch } from 'vue';
+import Chart from 'primevue/chart';
+import { ChartConfig } from '@/types/SimulateConfig';
+
+const props = defineProps<{
+	riskResults: any;
+	targetVariable: string;
+	chartConfig: ChartConfig;
+	size?: { width: number; height: number };
+}>();
+
+const chartSize = computed(() => {
+	if (props.size) return props.size;
+	return { width: 390, height: 190 };
+});
+
+const chartOptions = ref();
+const chartData = ref();
+const binCount = 6;
+
+const setChartOptions = () => ({
+	indexAxis: 'y',
+	responsive: false,
+	devicePixelRatio: 4,
+	maintainAspectRatio: false,
+	pointStyle: false,
+	animation: {
+		duration: 0
+	},
+	showLine: true,
+	plugins: {
+		legend: {
+			display: false
+		}
+	},
+	scales: {
+		x: {
+			title: {
+				display: true,
+				text: 'Number of Samples'
+			},
+			ticks: {
+				color: '#aaa',
+				maxTicksLimit: 5,
+				includeBounds: true,
+				// this rounds the tick label to nearest int
+				callback: (num) => num
+			},
+			grid: {
+				color: '#fff',
+				borderColor: '#fff'
+			}
+		},
+		y: {
+			title: {
+				display: true,
+				text: props.targetVariable
+			},
+			ticks: {
+				color: '#aaa',
+				includeBounds: true,
+				precision: 4
+			},
+			grid: {
+				color: '#fff',
+				borderColor: '#fff'
+			}
+		}
+	}
+});
+
+const getBinData = (data: number[]) => {
+	data.sort();
+	const minValue = data[0]; // data.getMin
+	const maxValue = data[data.length - 1]; // data.getMax
+	const stepSize = (maxValue - minValue) / (binCount - 1);
+	const bins: number[] = Array<number>(binCount).fill(0);
+	const binLabels: string[] = [];
+	for (let i = 0; i < binCount; i++) {
+		binLabels.push(
+			`${(minValue + stepSize * i).toFixed(4).toString()} - ${(minValue + stepSize * (i + 1))
+				.toFixed(4)
+				.toString()}`
```

Comment:
`toFixed(...)` is already returning a string, you don't need the extra `toString(...)`